### PR TITLE
fix: build scripts

### PR DESCRIPTION
**Pull request by andyMrtnzP**
- Removes the `build:selectors` script.
- Fixes schema generation for Windows
- Improves the searching of exported interfaces for the SDK. We used to just see if the string `"QuerySchema"` or `"BodySchema"` or `"ResponseSchema"` in the file's content. It could potentially create false-positives and try to parse files that have any exported propierties having that named, or comments. Now we use TypeScript's parser itself.
- Adds `ignoreHTTPSErrors` property to the `QuerySchema` interface in the CDP JSON schema (fixes https://github.com/browserless/browserless/issues/4020).

### CODE REVIEW

File: src/shared/chromium.ws.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import {
 import { Duplex } from 'stream';
 
 export interface QuerySchema extends SystemQueryParameters {
+  ignoreHTTPSErrors?: boolean;
```
Comment (joelgriffith) [inline src/shared/chromium.ws.ts:15]: I believe this is part of the launch opts, no? Shim _should_ move it into the right place if so and we shouldn't need this here?

**Review by joelgriffith — CHANGES_REQUESTED** [review]
Need clarification on the `ignoreHTTPSErrors` addition in chromium.ws.ts

File: src/shared/chromium.ws.ts (hunk 1 above)
Comment (andyMrtnzP) [reply src/shared/chromium.ws.ts:15]: It's not part of [Puppeteer's launch options](https://pptr.dev/browsers-api/browsers.launchoptions/#properties) :/. Doing requests using it throws an error:

```
Validating route query-params with QUERY schema
WebSocket query-params contain errors sending 400:"ignoreHTTPSErrors" is not allowed
```

Comment (joelgriffith) [reply src/shared/chromium.ws.ts:15]:
```import puppeteer from 'puppeteer-core';

const launch = {
  headless: 'boohoo',
  ignoreHTTPSErrors: true,
}

const job = async () => {
  try {
    const browser = await puppeteer.connect({
      browserWSEndpoint: `ws://localhost:3000/chromium?launch=${JSON.stringify(launch)}`,
    });

    console.log(await browser.version());
    const page = await browser.newPage();
    await page.goto('https://example.com');
    await page.close();
    await browser.disconnect();
  } catch (e) {
    console.log('Error', e);
  }
};

Array.from({ length: 1 }).map(job);```

Comment (joelgriffith) [reply src/shared/chromium.ws.ts:15]: Just to confirm after digging into their code: they don't have it as an explicit launchOption, however, once they launch and connect then they pass in this value when they connect. So it is part of their launchOptions but usage of that value is deferred until they have connected.

**Review by joelgriffith — APPROVED** [review]

**Pull request merged**